SYCL/Regression/context_is_destroyed_after_exception.cpp

```diff
@@ -0,0 +1,30 @@
+// REQUIRES: gpu
+
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_PI_TRACE=2 %GPU_RUN_PLACEHOLDER %t.out 2>&1 %GPU_CHECK_PLACEHOLDER
+
+#include <CL/sycl.hpp>
+
+int main() {
+  const auto GlobalRange = 1;
+  const auto LocalRange = 2;
+
+  sycl::queue myQueue{sycl::gpu_selector{}, [](sycl::exception_list elist) {
+                        for (auto e : elist)
+                          std::rethrow_exception(e);
+                      }};
+
+  try {
+    myQueue.parallel_for<class myclass>(
+        sycl::nd_range<1>{sycl::range<1>(GlobalRange),
```

Comment:
Could you please add comment about exception which is going to be generated? It takes me some time to notice that local range is greater that global range.

Reply:
Done